README.md

```diff
@@ -83,46 +83,43 @@ The documentation is generated using [Jazzy](https://github.com/realm/jazzy).
 ### Dynamic Framework Installation [⤴](#table-of-contents)
 
 1. Add **Buy.framework** target as a dependency:
-  1. Open **Your Project**, and then select your target.
-  2. Navigate to **Build Phases** > **Target Dependencies**.
-  3. Add `Buy.framework`.
+    1. Navigate to **Build Phases** > **Target Dependencies**.
```

Comment:
do you need to include a step to include the Buy.xcodeproj in the users project? Its not apparent here